### PR TITLE
Update calls to suppress build warnings

### PR DESCRIPTION
**Pull request by Shashi456**

### CODE REVIEW

File: Transformer/Operators.swift [hunk 1]
```diff
@@ -27,8 +27,7 @@ func gelu<Scalar: TensorFlowFloatingPoint>(_ x: Tensor<Scalar>) -> Tensor<Scalar
 /// Performs batched matrix multiplication of two tensors. The last two axes of each tensor
 /// are treated as the matrix axes; all others are treated as batch axes.
 @differentiable(
-    wrt: (left, right),
-    vjp: _vjpBatchedMatmul
+    wrt: (left, right)
```
Comment (dan-zheng) [inline Transformer/Operators.swift]: Please make this `@differentiable` attribute one line. It should be less than 100 columns.

**Comment by Shashi456** [comment]
@dan-zheng done.

**Comment by shabalind** [comment]
LGTM, thanks @Shashi456 ! 👍

File: Transformer/Operators.swift [hunk 2]
```diff
@@ -26,11 +26,7 @@ func gelu<Scalar: TensorFlowFloatingPoint>(_ x: Tensor<Scalar>) -> Tensor<Scalar
 
 /// Performs batched matrix multiplication of two tensors. The last two axes of each tensor
 /// are treated as the matrix axes; all others are treated as batch axes.
-@differentiable(
-    wrt: (left, right),
-    vjp: _vjpBatchedMatmul
-    where Scalar : Differentiable & TensorFlowFloatingPoint
-)
+@differentiable(wrt: (left, right) where Scalar : Differentiable & TensorFlowFloatingPoint)
```
Comment (dan-zheng) [inline Transformer/Operators.swift]: minor, feel free to address later: no space before colon in conformances. 
```suggestion
@differentiable(wrt: (left, right) where Scalar: Differentiable & TensorFlowFloatingPoint)
```

**Pull request merged**